Build error on Windows

I tried to run the installation of ContentaJS, which depends on plugnplay@3.5.0 on Windows.
During "npm install" execution I'm getting the following error.

> Try 'rm --help' for more information.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! plugnplay@3.5.0 build: `rm -fr lib; flow-remove-types --all src/ --out-dir lib/ && copy ./src/**/plugnplay.yml lib && flow-copy-source src lib`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the plugnplay@3.5.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\<----removed------>\Roaming\npm-cache\_logs\2019-02-05T13_33_45_149Z-debug.log
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! plugnplay@3.5.0 postinstall: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the plugnplay@3.5.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\<----removed------>\AppData\Roaming\npm-cache\_logs\2019-02-05T13_33_55_995Z-debug.log

Looks like that `plugnplay` is not using the `rimraf` module but  the UNIX `rm` command also on Windows.

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "semantic-release": "semantic-release",
     "postinstall": "npm run build",
     "flow": "flow",
-    "build": "rm -fr lib; flow-remove-types --all src/ --out-dir lib/ && copy ./src/**/plugnplay.yml lib && flow-copy-source src lib",
+    "build": "rimraf lib && flow-remove-types --all src/ --out-dir lib/ && flow-copy-source src lib",
     "lint": "eslint src",
     "emdaer": "emdaer"
   },
@@ -66,7 +66,8 @@
     "marked": "^1.1.1",
     "md5": "^2.2.1",
     "pify": "^4.0.0",
-    "require-subvert": "^0.1.0"
+    "require-subvert": "^0.1.0",
+    "rimraf": "^3.0.2"
   },
   "release": {
     "analyzeCommits": "semantic-release-conventional-commits"

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "semantic-release": "semantic-release",
     "postinstall": "npm run build",
     "flow": "flow",
-    "build": "rimraf lib && flow-remove-types --all src/ --out-dir lib/ && flow-copy-source src lib",
+    "build": "rimraf lib && flow-remove-types --all src/ --out-dir lib/ && copy ./src/**/plugnplay.yml lib && flow-copy-source src lib",
     "lint": "eslint src",
     "emdaer": "emdaer"
   },

